### PR TITLE
Add encryption support for importer

### PR DESCRIPTION
**Pull request by DorianZheng**
## What have you changed? (mandatory)
Add encryption support for importer.

## What are the type of the changes? (mandatory)
- New feature

## How has this PR been tested? (mandatory)
unit test

## Does this PR affect documentation (docs) update? (mandatory)
No

## Does this PR affect tidb-ansible update? (mandatory)
No

### CODE REVIEW

File: src/import/engine.rs [hunk 1]
```diff
@@ -348,22 +362,36 @@ mod tests {
 
     #[test]
     fn test_sst_writer() {
-        test_sst_writer_with(1, &[CF_WRITE]);
-        test_sst_writer_with(1024, &[CF_DEFAULT, CF_WRITE]);
+        test_sst_writer_with(1, &[CF_WRITE], &SecurityConfig::default());
+        test_sst_writer_with(1024, &[CF_DEFAULT, CF_WRITE], &SecurityConfig::default());
+
+        let path = TempDir::new("/tmp/encrypted_env_from_cipher_file").unwrap();
```
Comment (zhangjinpeng1987) [inline src/import/engine.rs]: Better to use a function `create_security_cfg` to wrapper L368-L373.

File: src/import/engine.rs [hunk 2]
```diff
@@ -348,22 +362,36 @@ mod tests {
...
 
-    fn test_sst_writer_with(value_size: usize, cf_names: &[&str]) {
+    fn test_sst_writer_with(value_size: usize, cf_names: &[&str], security_cfg: &SecurityConfig) {
         let temp_dir = TempDir::new("_test_sst_writer").unwrap();
 
         let cfg = DbConfig::default();
-        let db_opts = cfg.build_opt();
+        let mut db_opts = cfg.build_opt();
+        if !security_cfg.cipher_file.is_empty() {
+            let cipher_file = &security_cfg.cipher_file;
```
Comment (zhangjinpeng1987) [inline src/import/engine.rs]: Please combine L384 and L385.

**Comment by zhangjinpeng1987** [comment]
@DorianZheng CI failed 
test import::engine::tests::test_sst_writer ... thread 'import::engine::tests::test_sst_writer' panicked at 'called `Result::unwrap()` on an `Err` value: "failed to load cipher file: Os { code: 2, kind: NotFound, message: \"No such file or directory\" }"', src/libcore/result.rs:1009:5

**Comment by DorianZheng** [comment]
@zhangjinpeng1987 PTAL

**Pull request merged**